crates/sui-adapter-transactional-tests/tests/enums/enum_with_key_only.snap

```diff
@@ -14,12 +14,12 @@ Error: INVALID TEST. Unknown object, object(1,0)
 
 task 3, line 41:
 //# run test::f::transfer
-Error: Error checking transaction input objects: DependentPackageNotFound { package_id: 0x0000000000000000000000000000000000000000000000000000000000000000 }
+Error: Error checking transaction input objects: DependentPackageNotFound { package_id: 0x0 }
```

Comment:
And just to confirm, are these snapshot changes just informational, or affecting compatibility in some way?

Reply:
Yeah these are just because we now debug print them differently